deploy/deploy-centralvenv.sh

```diff
@@ -0,0 +1,1000 @@
+                                     (in double quotes and space separated e.g. "rqmgr2 reqmgr2ms")
+                                     (pip based package version syntax is also acceptable e.g. "wmcore==2.0.0")
+      -i <pypi-index>                The pypi index to use (i.e. prod or test) [Default: prod - pointing to https://pypi.org/simple/]
+      -h <help>                      Provides help to the current script
+
+    # Example: Deploy WMCore central services version 2.0.3rc1 linked with 'cmsweb-test1.cern.ch' as a frontend from 'test' pypi index
+    #          at destination /data/tmp/WMCore.venv3/ and using 'Some security string' as a security string for operationss at runtime:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -i test -l wmcore==2.0.3rc1 -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Same as above, but do not cleanup deployment area and reuse it from previous installtion - usefull for testing behaviour
+    #          of different versions during development or mix running from source and from pypi installed packages.
+    #          NOTE: The 'current' link must point to the proper deployment area e.g. either to 'srv/master' for running from source
+    #                or to 'srv/2.0.3rc1' for running from pypi installed package):
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -n -i test -l wmcore==2.0.3rc1 -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Deploy WMCore central services from source repository, use tag 2.0.0.pre3, linked with 'cmsweb-test1.cern.ch' as a frontend
+    #          at destination /data/tmp/WMCore.venv3/ and using 'Some security string' as a security string for operationss at runtime:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -s -t 2.0.0.pre3 -p "10003 9998" -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Same as above, but assume 'Yes' to all questions. To be used in order to chose the default flow and rely only
+    #          on the pameters set for configuring the deployment steps. This will avoid human intervention during deployment:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -y -s -t 2.0.0.pre3 -p "10003 9998" -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+
+    # DEPENDENCIES: All WMCore packages have OS or external libraries/packages dependencies, which are not having a pypi equivalent.
+    #               So far those has been resolved through the set of *.spec files maintained at: https://github.com/cms-sw/cmsdist/tree/comp_gcc630
+    #               Here follows the list of all direct (first level) dependencies per service generated from those spec files:
+
+    #               acdcserver: [python3==3.8.2, rotatelogs==2.2.25, couchdb16==1.6.1]
+    #               reqmgr2ms : [python3==3.8.2, jemalloc==5.2.0, rotatelogs==2.2.25, mongo==3.2.10]
```

Comment:
jemalloc is only required for optimization but it is not really required to run reqmgr2ms and other services. And, it should not be required since in dev mode I don't need it to run python server.

Reply:
Well for development purposes `jemalloc` may not be needed, but this is needed for the service to be run in reality. And If I understand correctly, at some point once we are ready to completly leave behind the RPM based setup, I  think we are about to use this script to deploy the services inside the container in order to have equivalent runtime and development environment. So these were the reasons to put it there. And also, without it, the so generated `manage` scripts per service are simply not usable. See line 771 in the current version:
https://github.com/dmwm/WMCore/blob/159835d434a34cb3a060d9387d8ce3c4f28917af/deploy/deploy-centralvenv.sh#L771

Reply:
No, I totally disagree, jemalloc is optional, I can successfully run `wmc-httpd` without it. What you need to do is to patch manage scripts to test jemalloc, if it exists you can use it, otherwise it should be empty string, e.g.
```
jemalloc=`command -v jemalloc`
...
$jemalloc wmc-httpd ...
```

Reply:
Well this is a nice trick. I like it.